### DISCUSSION TITLE
svgr with @storybook/nextjs within a Nx monorepo

### DISCUSSION THREAD

**Discussion by ganlhi** [storybookjs/storybook#26404] [Help]
### Summary

Hello! 

I have a library in a Nx monorepo, in which I use `@storybook/nextjs`. The components I'm working on in there import other components that use svg as React components (for icons). It works fine in my app and in a Storybook instance using `@storybook/react-vite`, but here I cannot make it work. 

I get the following kind of errors:
```
./packages/flamingo/src/lib/icon/svg-components.ts 106:0-69export 'ReactComponent' (reexported as 'Basictrash') was not found in './svg/basic/trash.svg' (possible exports: default)
```

I tried a lot of things I found online, involving `webpackFinal`, like that:
```
  webpackFinal: async cfg => {
    cfg?.module?.rules?.forEach(rule => {
      if (!rule || typeof rule !== 'object') return;
      if (rule.test instanceof RegExp && rule.test.test('.svg')) {
        rule.exclude = /\.svg$/;
      }
    });
    cfg.module?.rules?.push({
      test: /\.svg$/,
      use: ['@svgr/webpack'],
    });
    return cfg;
  }
```
But no luck...

**Comment by valentinpalkovic** [comment]
Hi @ganlhi 

Does this comment help?
https://github.com/storybookjs/storybook/issues/18557#issuecomment-1426150038

**Reply by ganlhi** [reply]
Hi! Unfortunately not, I still have the same error.

**Reply by valentinpalkovic** [reply]
Could you please provide a reproduction here: https://storybook.new. Alternatively, you can also share a minimal reproduction repo.

**Reply by ganlhi** [reply]
Here is a repository to reproduce the issue: https://github.com/ganlhi/sb-issue

- Run `npm install`
- Run `npm start`
- Go to http://localhost:4400
- The story should show a lock icon, but it does not, and the error message is visible in the browser console.

**Reply by ganlhi** [reply]
Hello there! Did you have a chance to look at my reproduction?
